File: solucoes/2021/nivel jr/fase 3/py/ogro.py
M = int(input())
direita = ''
esquerda = ''

for i in range(M):
    if(i < 5):
        direita += 'I'
    else:
        esquerda += 'I'

if(M == 0):
    print('*', '*')
elif(M <= 5):
    print(direita, '*')
else:
    print(direita, esquerda)

File: solucoes/2021/nivel jr/fase 3/js/ogro.js
var M
var direita = ''
var esquerda = ''

scanf("%d", "M")

for(var i = 0; i < M; i++){
    if(i < 5){
        direita += 'I'
    }
    else{
        esquerda += 'I'
    }
}

if(M == 0){
    printf("%s\n%s", '*', '*')
}
else if(M <= 5){
    printf("%s\n%s", direita, '*')
}
else{
    printf("%s\n%s", direita, esquerda)
}